server/routerlicious/packages/routerlicious-base/src/nexus/runner.ts

```diff
@@ -0,0 +1,222 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import cluster from "cluster";
+import { Deferred, TypedEventEmitter } from "@fluidframework/common-utils";
+import {
+	ICache,
+	IClientManager,
+	IDocumentStorage,
+	IOrdererManager,
+	IRunner,
+	ITenantManager,
+	IThrottler,
+	IThrottleAndUsageStorageManager,
+	IWebServer,
+	IWebServerFactory,
+	ITokenRevocationManager,
+	IWebSocketTracker,
+	IRevokedTokenChecker,
+} from "@fluidframework/server-services-core";
+import { Provider } from "nconf";
+import * as winston from "winston";
+import { createMetricClient } from "@fluidframework/server-services";
+import { LumberEventName, Lumberjack, LogLevel } from "@fluidframework/server-services-telemetry";
+import {
+	configureWebSocketServices,
+	ICollaborationSessionEvents,
+} from "@fluidframework/server-lambdas";
+import { runnerHttpServerStop } from "@fluidframework/server-services-shared";
+import * as app from "./app";
+
+export class NexusRunner implements IRunner {
+	private server: IWebServer;
+	private runningDeferred: Deferred<void>;
+	private stopped: boolean = false;
+	private readonly runnerMetric = Lumberjack.newLumberMetric(LumberEventName.NexusRunner);
+
+	constructor(
+		private readonly serverFactory: IWebServerFactory,
+		private readonly config: Provider,
+		private readonly port: string | number,
+		private readonly orderManager: IOrdererManager,
+		private readonly tenantManager: ITenantManager,
+		private readonly socketConnectTenantThrottler: IThrottler,
+		private readonly socketConnectClusterThrottler: IThrottler,
+		private readonly socketSubmitOpThrottler: IThrottler,
+		private readonly socketSubmitSignalThrottler: IThrottler,
+		private readonly storage: IDocumentStorage,
+		private readonly clientManager: IClientManager,
+		private readonly metricClientConfig: any,
+		private readonly throttleAndUsageStorageManager?: IThrottleAndUsageStorageManager,
+		private readonly verifyMaxMessageSize?: boolean,
+		private readonly redisCache?: ICache,
+		private readonly socketTracker?: IWebSocketTracker,
+		private readonly tokenRevocationManager?: ITokenRevocationManager,
+		private readonly revokedTokenChecker?: IRevokedTokenChecker,
+		private readonly collaborationSessionEventEmitter?: TypedEventEmitter<ICollaborationSessionEvents>,
+	) {}
+
+	// eslint-disable-next-line @typescript-eslint/promise-function-async
+	public start(): Promise<void> {
+		this.runningDeferred = new Deferred<void>();
+
+		// Create the HTTP server and attach nexus to it
+		const nexus = app.create(this.config, this.collaborationSessionEventEmitter);
```

Comment:
We don't need an Express app for Nexus. This can be replaced with a basic request listener that responds with 200 OK

```ts
(req, res) => {
  res.writeHead(200, {});
  res.write("OK");
  res.end();
}
```